test_UidGenerator.lua

```diff
@@ -0,0 +1,4 @@
+local sf = require "snowflake"
+
+sf.init(0x1f, 0x1f, nil, 11, 11, 13, 1000)
```

Comment:
We can put it into test_sonyflake.lua, and we need to check if the delta option works.